webservice/pom.xml

```diff
@@ -12,7 +12,7 @@
     <artifactId>registry-webservice</artifactId>
 
     <properties>
-        <dropwizard.metrics.version>3.1.2</dropwizard.metrics.version>
+        <dropwizard.metrics.version>4.0.5</dropwizard.metrics.version>
         <dropwizard.swagger.version>1.2.4-1</dropwizard.swagger.version>
```

Comment:
as per the compatibility matrix of this dw swagger (https://github.com/smoketurner/dropwizard-swagger#version-matrix)
it should be updated too to 1.3.17-1  is the latest 1.3.x

Reply:
Thanks, I didn't notice this compatibility matrix.